src/Microsoft.DotNet.Build.Tasks/Microsoft.DotNet.Build.Tasks.csproj

```diff
@@ -82,9 +83,81 @@
   </ItemGroup>
   <ItemGroup>
     <None Include="project.json" />
-    <None Include="PackageFiles\**\*.*" />
+    <None Include="PackageFiles\ApiCompat.runtimeconfig.json" />
```

Comment:
I don't personally mind breaking the `**\*.*` into its parts, since I use VS, but was this intentional?